### PR TITLE
Notes

### PR DESCRIPTION
**Pull request by nonso-uj**

### CODE REVIEW

**Review by vokecodes — CHANGES_REQUESTED** [review]

File: src/pages/chef/ChefDineIn.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { RESTAURANT_ORDER_URL } from "../../_redux/urls";
 import { SERVER } from "../../config/axios";
 import { Modal } from "@mui/material";
 import { IoMdClose } from "react-icons/io";
-import InfinityScroll from "../../components/InfinityScroll";
+import InfiniteScroll from "react-infinite-scroll-component";
```
[the remark this answers is not in the document]
Comment (nonso-uj) [reply src/pages/chef/ChefDineIn.tsx:18]: Oh, thought it was a typo, the glitch stopped when I changed it, I'll change it back

[the remark this answers is not in the document]
Comment (vokecodes) [reply src/pages/chef/ChefDineIn.tsx:18]: No, fix with the component use

File: src/pages/chef/Kitchen.tsx [hunk 2]
```diff
@@ -780,21 +780,18 @@ const Kitchen = () => {
 
       <div className="w-fit md:w-full md:px-6 py-4">
         <InfiniteScroll
-          dataLength={restaurantOrders?.length} // This is important to track the length of your data array
```
Comment (vokecodes) [inline src/pages/chef/Kitchen.tsx:783]: @nonso-uj Make us of the component InfinityScroll

**Review by vokecodes — CHANGES_REQUESTED** [review]

File: src/pages/chef/Kitchen.tsx (hunk 2 above)
Comment (vokecodes) [reply src/pages/chef/Kitchen.tsx:783]: @nonso-uj I can still see it here

**Review by vokecodes — CHANGES_REQUESTED** [review]

**Comment by vokecodes** [comment]
@nonso-uj There are conflicts

**Pull request merged**